### DISCUSSION TITLE
一个奇怪的问题

### DISCUSSION THREAD

**Discussion by mailuser963** [zdz/ServerStatus-Rust#183] [Q&A]
我在debian10的vps上部署了stat_server服务。
在我的其他vps上部署了stat_client都没有报错。
但是我的列表里只能显示两台debian10系统的vps，还有两台centOS的一直是失联状态。
重新安装那两台centOS的client也都没有报错，就是无法显示到列表中。
问题可能会出在什么地方呢。。。
多谢

**Comment by zdz** [comment]
是不是id(name)重复了？

**Reply by mailuser963** [reply]
找到原因了，客户端用快速部署脚本带上vnstat=1参数后，脚本安装vnstat后没有初始化制定监听网卡，vnstat运转不正常，连带着导致stat_client服务启动失败。但是快速部署脚本并没有报错，看上去一切正常。我是去查看了系统stat_client服务的状态才发现是启动失败，然后用排除法定位到是vnstat的问题。如果不使用快速部署应该就能看到服务启动的报错了。手动先配置好vnstat后，快速部署就一切正常了。
